src/modules/interactive.js

```diff
@@ -0,0 +1,35 @@
+import { toDoListData } from './todo.js';
+import resetIndex from './resetIndex.js';
+import { setData } from './setItems.js';
+
+export function removeTask(reindex) {
+  const toDoListDatas = toDoListData().filter((todo) => todo.index !== reindex);
+  resetIndex(toDoListDatas);
+  setData(toDoListDatas);
+}
+
+export function editTodo(reindex, todoDescription) {
+  const toDoListDatas = toDoListData().map((todo) => {
+    if (todo.index === reindex) {
+      return { ...todo, description: todoDescription };
+    }
+    return todo;
+  });
+  setData(toDoListDatas);
+}
+
+export function completeTodo(reindex) {
+  const duplicate = toDoListData();
+  duplicate.forEach((element) => {
+    if (element.index === reindex) {
+      element.completed = !element.completed;
+    }
+  });
+  setData(duplicate);
+}
+
+export function clearCompletedTodos() {
+  const toDoListDatas = toDoListData().filter((todo) => !todo.completed);
+  resetIndex(toDoListDatas);
+  setData(toDoListDatas);
+}
```

Comment:
The "clear all completed" tasks button is not working. Currently, nothing happens when I click it.

- Kindly look into it and make sure that it removes all completed tasks from the array when the button is clicked.